posix.mak

```diff
@@ -176,6 +176,9 @@ $(DOCDIR)/core_sys_darwin_netinet_%.html : src/core/sys/darwin/netinet/%.d $(DMD
 $(DOCDIR)/rt_%.html : src/rt/%.d $(DMD)
 	$(DMD) $(DDOCFLAGS) -Df$@ project.ddoc $(DOCFMT) $<
 
+$(DOCDIR)/rt_array_array.html : src/rt/array/array.d $(DMD)
```

Comment:
I think you mean `src/rt/array/append.d`.

Reply:
Thanks :)